jupyterhub-training-notebooks/basics.ipynb

```diff
@@ -0,0 +1,185 @@
+    "\n",
+    "<img src=\"../images/run_selected_cells.png\" alt=\"Drawing\" style=\"width: 700px;\"/>\n",
+    "\n",
+    "Or select the cell you would like to run and click the play button at the top of the main work area.\n",
+    "\n",
+    "<img src=\"../images/play_cell.png\" alt=\"Drawing\" style=\"width: 700px;\"/>\n",
+    "\n",
+    "Pick your favourite method and have a go at running the cell below."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "echo \"Hello World! Congratulations, you got your first cell to run\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## How to tell when things are running \n",
+    "When things are running a `*` symbol will appear in the square brackets to the left of the cell. \n",
+    "\n",
+    "<img src=\"../images/cell_running.png\" alt=\"Drawing\" style=\"width: 700px;\"/>\n",
+    "\n",
+    "When this has finished running it will change to a number.\n",
+    "\n",
+    "<img src=\"../images/cell_completed_running.png\" alt=\"Drawing\" style=\"width: 700px;\"/>\n",
+    "\n",
+    "Run the below cell to see this work. It should run for 5 seconds."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sleep 5"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Adding extra cells\n",
+    "To add in extra cells click the `+` button at the top of the jupyternotebook tab. \n",
+    "\n",
+    "<img src=\"../images/add_cell.png\" alt=\"Drawing\" style=\"width: 700px;\"/>\n",
+    "\n",
+    "Try adding a new cell below"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Removing cells \n",
+    "To remove a cell simply click on it and click the scissors at the top of jupyternotebook tab. \n",
+    "\n",
+    "<img src=\"../images/cut_cell.png\" alt=\"Drawing\" style=\"width: 700px;\"/>\n",
+    "\n",
+    "Try removing the cell below \n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Opening a terminal\n",
+    "To open a new terminal you can simply select '_terminal_' from the launcher. If you don't have a launcher open you can select _File -> New -> Terminal_ from the top menu bar.\n",
+    "\n",
+    "<img src=\"../images/new_terminal.png\" alt=\"Drawing\" style=\"width: 700px;\"/>\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Clearing the output from your notebook \n",
+    "When you are running cells the output will be printed to the notebook. To clear this and any variables that are stored click _Edit -> Clear all Outputs_ in the top bar.\n",
```

Comment:
may be worth pointing out that this clears the cells but will not reverse anything done to files (creation, modification or deletion)